src/de.rs

```diff
@@ -165,12 +169,9 @@ impl<'de, 'a, R: dec::Read<'de>> serde::Deserializer<'de> for &'a mut Deserializ
             major::STRING => de.deserialize_string(visitor),
             major::ARRAY => de.deserialize_seq(visitor),
             major::MAP => de.deserialize_map(visitor),
-            // NOTE: that this does not support untagged enum.
-            // see https://github.com/serde-rs/serde/issues/1682
             major::TAG => match byte {
-                0xc2 => de.deserialize_u128(visitor),
```

Comment:
are you removing support for 128 bit numbers ?

Reply:
Yes, they have a different tag => not valid DAG-CBOR.